### PR TITLE
Onboarding fix archived page

### LINKED ISSUE #779: Refactor archived user page
### Issue Description

Currently, the archived user page is not good needs to be refactored and provide Ankush Twitter

### Expected Behavior

Need to add twitter handle of Ankush 

### Current Behavior

Only plain text archived user is shown

### Screenshots

image

### Reproducibility

 - [ ] This issue is reproducible
-  [x] This issue is not reproducible

### Steps to Reproduce
1. [Step 1]
2. [Step 2]

### Severity/Priority
 - [ ] Critical
 - [x] High
 - [ ] Medium
 - [ ] Low

### Additional Information

[Include any additional context, logs, or details that might be helpful in addressing the issue.]

### Checklist

- [x] I have read and followed the project's code of conduct.
- [x] I have searched for similar issues before creating this one.
- [x] I have provided all the necessary information to understand and reproduce the issue.
- [x] I am willing to contribute to the resolution of this issue.

---

Thank you for contributing to our project! We appreciate your help in improving it.

### PR DESCRIPTION
**Pull request by vinit717**
### What is the change?

refactor the archived page design for good UX

##### Example :

- Done this
- Made these changes

### Is Development Tested?

- [ ] Yes
- [ ] No

### Before / After Change Screenshots

For visual or interaction changes. Please provide a video / screenshot.

##### Example:

Before :
image

After :
image
image)

**Note :**
What to delete and what not to

- Keep the headings and answer the questions.
- Delete the examples
- Delete the description below headings `What is the change?` & `Before/After Screenshots`

### CODE REVIEW

File: app/controllers/join.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ export default class JoinController extends Controller {
   @tracked chaincode = 'Generate chaincode';
   @tracked isChaincodeClicked = false;
   @tracked isLoading = false;
+  ANKUSH_TWITTER = ANKUSH_TWITTER;
```
Comment (satyam73) [inline app/controllers/join.js:15]: This re-declaration of same variable is not needed we can skip and directly use the imported one

File: app/templates/join.hbs [hunk 2]
```diff
@@ -1,25 +1,30 @@
...
-          @isChaincodeClicked={{this.isChaincodeClicked}}
-          @isLoading={{this.isLoading}}
-          @handleGenerateChaincode={{this.handleGenerateChaincode}}
-        />
-      {{else}}
-        <Stepper />
-      {{/if}}
-    {{/if}}
+  {{#if this.login.userData.roles.archived}}
+<OnboardingCard data-test-onboarding-card>
```
Comment (satyam73) [inline app/templates/join.hbs:9]: is this data-test attribute works?

Comment (satyam73) [reply app/templates/join.hbs:9]: also can we use getter function here for better readability?

File: tests/integration/components/onboarding-card-test.js [hunk 3]
```diff
@@ -11,11 +11,32 @@ module('Integration | Component | card', function (hooks) {
...
+      .dom('[data-test-archived-message]')
+      .hasText(
+        'This user account has been archived. If you have any questions or need assistance, please contact Ankush via Twitter.',
+      );
+    assert
+      .dom(
+        '[data-test-archived-contact-container] [data-test-archived-contact-link]',
+      )
+      .hasText('Contact Ankush on Twitter');
   });
```
Comment (satyam73) [inline tests/integration/components/onboarding-card-test.js:41]: Can we do it like this?
```suggestion
    assert
      .dom(
        '[data-test-archived-contact-link]',
      )
      .hasText('Contact Ankush on Twitter');
  });
```

**Review by MehulKChaudhari — APPROVED** [review]

**Review by MehulKChaudhari — DISMISSED** [review]
Approving, because everything looks good after making changes of requested by Satyam.

**Comment by MehulKChaudhari** [comment]
can we add a little margin on the left and right side in mobile? may be 10-15px. that will look better

**Review by lakshayman — APPROVED** [review]

File: app/controllers/join.js (hunk 1 above)
Comment (satyam73) [reply app/controllers/join.js:15]: resolving this since it's been used in hbs file so this is needed

**Review by satyam73 — DISMISSED** [review]
LGTM

**Comment by vinit717** [comment]
> can we add a little margin on the left and right side in mobile? may be 10-15px. that will look better

No, as the mobile screen is small and if we make the card more small test alignment and overall look won't good at all

**Review by satyam73 — APPROVED** [review]

**Pull request merged**